Yoast/yoast-test-helper
# Upgraded grunt-contrib-compress in order to upgrade tar-fs

Test instructions:

Run `grunt artifact`.
Install the plugin as a zip `(archive.zip)` and make sure it works.

Fixes #38
